requirements/reproducible-linux.txt

```diff
@@ -21,24 +21,6 @@ aws-sam-translator==1.53.0 \
     --hash=sha256:84d780ad82f1a176e2f5d4c397749d1e71214cc97ee7cccd50f823fd7c7e7cdf \
     --hash=sha256:85252646cf123642d08442137b60445e69e30bfd2f8b663b1202b20ab3782b10
     # via aws-sam-cli (setup.py)
-backports-zoneinfo==0.2.1 \
```

Comment:
As Wing mentioned above, these file should be generated with python3.7, otherwise we will miss some dependencies due to version difference.

Reply:
Sure, will change it